scripts/_cache.sh

```diff
@@ -30,7 +30,11 @@ function _cache() {
         "fill-front-end") _cache_fill_front_end $args ;;
         "fill-public-api") _cache_fill_public_api $args ;;
         "flush") _cache_flush $args ;;
-        
+
+        "flush-front-end-v2") _cache_flush_front_end_v2 $args ;;
```

Comment:
I haven't added these to the help text of the CLI module just yet because they're only to be used when caching v2 is switched on (which is off by default right now)